.github/workflows/ulwgl-python.yml

```diff
@@ -32,5 +33,10 @@ jobs:
         pip install ruff
         ruff --output-format github ulwgl_*.py
     - name: Test with unittest
+      if: matrix.python-version == 3.10
```

Comment:
I think this should be referenced as `${{ matrix.version }}` for it to work.

Reply:
Also maybe quote the string of the version number you compare with. Github actions run through js, and sometimes it gets weird.

Reply:
> I think this should be referenced as `${{ matrix.version }}` for it to work.

It's already working without it

Reply:
> Also maybe quote the string of the version number you compare with. Github actions run through js, and sometimes it gets weird.

Wow, didn't know that. So this?

> matrix.python-version == "3.10"

But if it's JS, then strictly equals would be preferred to avoid implicit type coercion?

Reply:
I had an issue some time ago where I was specifying the matrix like this
```
version: [3.9, 3.10, 3.11, 3.12]
```
instead of
```
version: ['3.9', '3.10', '3.11', '3.12']
```
and for some reason it would lead to interpreting the `3.10` version as `3.1` [link](https://github.com/RareDevs/Rare/actions/runs/7604206484)

I have trust issues towards GH Actions since then.

Reply:
To add on, it appears the GH Actions is also strict about quotes as well...

Reply:
> I think this should be referenced as `${{ matrix.version }}` for it to work.

Sorry, but you were right about this!